### PR TITLE
ci: add pana workflow

### PR DESCRIPTION
**Pull request by felangel**
## Status

**READY**

## Description

- ci: add pana workflow

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [X] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: .github/workflows/dart_frog.yaml [hunk 1]
```diff
@@ -21,3 +21,21 @@ jobs:
...
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - uses: dart-lang/setup-dart@v1
+
+      - name: Install Dependencies
+        run: |
+          dart pub get
+          dart pub global activate pana
+      - name: Verify Pub Score
+        run: ../../tool/verify_pub_score.sh
```
Comment (wolfenrain) [inline .github/workflows/dart_frog.yaml:41]: Right now it holds a single file, but in the context of naming things shouldn't it be plural? Tools?

**Review by wolfenrain — APPROVED** [review]

File: .github/workflows/dart_frog.yaml (hunk 1 above)
Comment (felangel) [reply .github/workflows/dart_frog.yaml:41]: I was going off of the naming convention in other dart projects haha https://github.com/dart-lang/pub/tree/400f21e9883ce6555b66d3ef82f0b732ba9b9fc8/tool

Comment (felangel) [reply .github/workflows/dart_frog.yaml:41]: Happy to adjust it though 👍

Comment (wolfenrain) [reply .github/workflows/dart_frog.yaml:41]: But in our naming conventions we prefer plurals right?🤔

**Pull request merged**